webapp/src/editortoolbar.tsx

```diff
@@ -266,17 +323,15 @@ export class EditorToolbar extends data.Component<ISettingsProps, {}> {
                         </div>
                         <div className="five wide column">
                             <div className="ui grid right aligned">
-                                {compileBtn && <div className="row">
-                                    <div className="column">
-                                        <EditorToolbarButton role="menuitem" className={`primary large fluid download-button download-button-full ${downloadButtonClasses}`} icon={downloadIcon} text={downloadText} title={compileTooltip} onButtonClick={this.compile} view='tablet' />
+                                {compileBtn && <div className="ui row items" style={ { paddingBottom: 0, marginBottom: '1rem' } }>
```

Comment:
Why the inline style here?

Reply:
Also on line 332 (though it looks like that one was there before, it should also probably be a class)

Reply:
this was an attempt to avoid having a custom css class/id specifically for the "not collapsed + tablet" view alone, i think in this case inline is a little nicer than a long selector but i'm fine with putting it in the css too